pkg/cli/admin/policy/modify_scc.go

```diff
@@ -231,105 +231,63 @@ func (o *SCCModificationOptions) CompleteGroups(f kcmdutil.Factory, cmd *cobra.C
 }
 
 func (o *SCCModificationOptions) AddSCC() error {
-	scc, err := o.SCCInterface.Get(context.TODO(), o.SCCName, metav1.GetOptions{})
-	if err != nil {
-		return err
-	}
-
-	users, groups := stringSubjectsFor(o.DefaultSubjectNamespace, o.Subjects)
-	usersToAdd, _ := diff(users, scc.Users)
-	groupsToAdd, _ := diff(groups, scc.Groups)
-
-	scc.Users = append(scc.Users, usersToAdd...)
-	scc.Groups = append(scc.Groups, groupsToAdd...)
-
-	message := successMessage(true, o.IsGroup, users, groups)
-
-	p, err := o.ToPrinter(message)
-	if err != nil {
-		return err
+	clusterRole := &rbacv1.ClusterRole{
+		// this is ok because we know exactly how we want to be serialized
+		TypeMeta:   metav1.TypeMeta{APIVersion: rbacv1.SchemeGroupVersion.String(), Kind: "ClusterRole"},
+		ObjectMeta: metav1.ObjectMeta{Name: fmt.Sprintf(RBACNamesFmt, o.SCCName)},
+		Rules:      []rbacv1.PolicyRule{rbacv1helpers.NewRule("use").Groups(security.GroupName).Resources("securitycontextconstraints").Names(o.SCCName).RuleOrDie()},
 	}
 
-	if o.DryRunStrategy == kcmdutil.DryRunClient {
-		return p.PrintObj(scc, o.Out)
+	if _, err := o.RbacClient.ClusterRoles().Get(context.TODO(), clusterRole.Name, metav1.GetOptions{}); err != nil && kapierrors.IsNotFound(err) {
+		p, err := o.ToPrinter("added")
+		if err != nil {
+			return err
+		}
+		if o.DryRunStrategy == kcmdutil.DryRunClient || (o.PrintFlags.OutputFormat != nil && len(*o.PrintFlags.OutputFormat) > 0) {
```

Comment:
no creating the clusterrole if an `OutputFormat` is specified?

Reply:
Good catch, I copied that from `modify_roles.go`, which probably needs an update too.